TreasureHunt/Assets/2.Scripts/MakerSceneManager.cs

```diff
@@ -16,22 +16,21 @@ void start()
 
 	public void uploadToCloud()
 	{
-		/*TODO(hyunSo): works on my laptop but not on my phone. Guess I should have change the camera methad as WebCam
```

Comment:
Yes I solved the problem! I find out that the problem is vuforia cloud do not allow file size bigger than 2MB, so camera method had no problem.